src/main/java/com/browserstack/automate/ci/common/tracking/PluginsTracker.java

```diff
@@ -1,24 +1,24 @@
 package com.browserstack.automate.ci.common.tracking;
 
+import com.browserstack.automate.ci.common.BrowserStackBuildWrapperOperations;
 import com.browserstack.automate.ci.common.Tools;
 import com.browserstack.automate.ci.common.constants.Constants;
-import okhttp3.Call;
-import okhttp3.Callback;
-import okhttp3.MediaType;
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.RequestBody;
-import okhttp3.Response;
+import com.browserstack.automate.ci.common.proxysettings.JenkinsProxySettings;
+import hudson.ProxyConfiguration;
+import okhttp3.*;
```

Comment:
verify for `*` imports at other places as well...

Reply:
Avoid `*` in java. A recommended convention.
There is no performance impact as such. Only if two classes have the same name, you have to specify a class with prefix namespace eg `Java.util.Date()` vs `Java.sql.Date() `
https://stackoverflow.com/questions/187453/import-package-vs-import-package-specifictype

Reply:
Done

Reply:
can be resolved...